tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,222 @@ keywords:
   - insertion sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Shivashish-rwt"/>
+
+## Overview
+
+Insertion sort is one of the sorting algorithms that sort the elements by placing an unsorted element in correct sorted order one at a time.
+
+First of all, let us understand how an sorted `array` (If you don't what an array is. Please refer [here](/tutorials/basic-topics/arrays.md)) looks like. Understanding an sorted array will give us the overview of the insertion sort. An sorted array is given below.
```

Comment:
You have given a reference for the array tutorials, but it's hard to know it has a reference link. 
Either bold it or underline, it will be easy to point out, hey here is a reference for you to check.

Reply:
Check if this is suitable:

First of all, let us understand how an sorted `array` (**[Here is a reference for you to check](../arrays.md)**) looks like. Understanding an sorted array will give us the overview of the insertion sort. An sorted array is given below.

It looks something like this:

![Output](https://user-images.githubusercontent.com/85662806/194005990-9da7c900-fca0-43bf-933d-5bb485683854.png)